src/Input.test.jsx

```diff
@@ -0,0 +1,38 @@
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import Input from './Input';
+
+import { TASK_TITLE } from './fixture';
+
+describe('Input', () => {
+  const onChangeTitle = jest.fn();
+  const onClickAddTask = jest.fn();
+
+  const renderInputComponent = () => render(
+    <Input value={TASK_TITLE} onChange={onChangeTitle} onClick={onClickAddTask} />,
+  );
+
+  it('input에 prop으로 넘겨준 value값이 입력되어있다', () => {
+    renderInputComponent();
+    const input = screen.getByRole('textbox');
+    expect(input.value).toBe(TASK_TITLE);
+  });
+
+  it('투두 인풋의 버튼을 클릭하면 onClickAddTask 이벤트가 호출된다', () => {
+    const { getByText } = renderInputComponent();
+
+    const addTodoButton = getByText('추가');
+    addTodoButton.click();
```

Comment:
오 이런 방법도 있었군요 ㅎㅎ fireEvent를 사용해보는 것도 좋을 것 같아요
```suggestion
    fireEvent.click(getByText('추가'));
```

Reply:
저도 IDE의 지원으로 처음 알게되었습니다 😅
다른 테스트에서 fireEvent를 쓰고 있으니 실험적으로 써보고 싶었던건데, 확실히 테스트를 위한 목적으로는 좀 더 다양한? 편의성을 지원해주는 fireEvent 혹은 userEvent가 좋겠더라구요. 
수정했습니다 :)